scripts/package/postinstall.sh

```diff
@@ -94,15 +94,28 @@ finish_permissions() {
   # We also change the owner of the binary to observiq-otel-collector
   chown -R observiq-otel-collector:observiq-otel-collector /opt/observiq-otel-collector/observiq-otel-collector /opt/observiq-otel-collector/plugins/*
```

Comment:
```suggestion
  chown -R observiq-otel-collector:observiq-otel-collector \
    /opt/observiq-otel-collector/observiq-otel-collector \
    /opt/observiq-otel-collector/opampsupervisor \
    /opt/observiq-otel-collector/plugins/*
```

Added `opampsupervisor` to be consistent. Technically its fine being owned by `root`, because it is executable by anyone.

```
-rwxr-xr-x 1 observiq-otel-collector observiq-otel-collector 262631424 Aug 13 18:55 observiq-otel-collector
-rwxr-xr-x 1 root                    root                     14960246 Aug 13 18:55 opampsupervisor
```